neurodamus/ngv.py

```diff
@@ -272,12 +277,15 @@ def finalize(self, astrocyte, base_Seed, *, base_connections=None, **kw):
         n_bindings = 0
         pc = Nd.pc
 
+        def ustate_event_handler2(syn_gid):
+            return lambda: print("GOOD netcon event 2. Spiking via v-gid: "+str(syn_gid))
+
         if GlobalConfig.debug_conn:
             if GlobalConfig.debug_conn == [self.tgid]:
                 logging.debug("Finalizing conn %s. N params: %d", self, len(self._synapse_params))
             elif GlobalConfig.debug_conn == [self.sgid, self.tgid]:
                 logging.debug("Finalizing conn %s. Params:\n%s", self, self._synapse_params)
-
+        print("Finalizing conn %s. " % (self))
```

Comment:
Can we use `logging.debug` instead of `print` so it will only be printed in debug mode ? Same for the other `print` msgs.